mypy/typeanal.py

```diff
@@ -1284,7 +1284,7 @@ def analyze_literal_param(self, idx: int, arg: Type, ctx: Context) -> list[Type]
                 out.extend(union_result)
             return out
         else:
-            self.fail(f"Parameter {idx} of Literal[...] is invalid", ctx)
+            self.fail(f"Parameter {idx} of Literal[...] is invalid", ctx, code=codes.VALID_TYPE)
```

Comment:
I can't comment there, but should the error on line 1253 above also have this code?